### PR TITLE
Run generate-templates as a part of make generate

### PR DESCRIPTION
**Pull request by Amulyam24**
**What this PR does / why we need it**:
This change will verify if the cluster templates  are updated when the generate target is run on `make verify`.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

/area provider/ibmcloud

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note
Run generate-templates as a part of make generate
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -155,7 +155,7 @@ help:  # Display this help
 # Generate code
 .PHONY: generate
 generate: ## Run all generate-go generate-modules generate-manifests generate-go-deepcopy generate-go-conversions
```
Comment (Prajyot-Parab) [inline Makefile:157]: Minor update the comment as well

Comment (Amulyam24) [reply Makefile]: Updated.

**Review by Prajyot-Parab — APPROVED** [review]
/lgtm

**Review by mkumatag — APPROVED** [review]
/lgtm

**Pull request merged**